test/quantization/core/experimental/test_quantized_tensor.py

```diff
@@ -1,19 +1,116 @@
 # Owner(s): ["oncall: quantization"]
 
 import torch
-from torch.ao.quantization.experimental.APoT_tensor import TensorAPoT
+from torch import quantize_per_tensor
+from torch.ao.quantization.experimental.APoT_tensor import TensorAPoT, APoTRepr
 import unittest
+import random
+quantize_APoT = TensorAPoT.quantize_APoT
+dequantize = TensorAPoT.dequantize
 
 class TestQuantizedTensor(unittest.TestCase):
-    def test_quantize_APoT(self):
-        t = torch.Tensor()
-        with self.assertRaises(NotImplementedError):
-            TensorAPoT.quantize_APoT(t)
+    r""" Tests quantize_APoT result (int representation) on random 1-dim tensor
+        and hardcoded values for b, k by comparing to uniform observer
+        (non-uniform observer reduces to uniform for k = 1)
+        quantized tensor (https://pytorch.org/docs/stable/generated/torch.quantize_per_tensor.html)
+        * tensor2quantize: Tensor
+        * b: 4
+        * k: 1
+    """
+    def test_quantize_APoT_rand_1d(self):
+        # generate random size of tensor2dequantize between 1 -> 16
+        # because there are 2**b = 2**4 quantization levels total
+        size = random.randint(1, 16)
```

Comment:
we can use torch.randint

Reply:
i'm using torch.rand to generate the tensors so i can get fp values, random.randint was to generate random sizes for the dimensions